scripts/lib/install

```diff
@@ -127,7 +127,7 @@ if [ "$VIRTUALENV_NEEDED" = "yes" ]; then
     "$ZULIP_PATH"/scripts/lib/create-production-venv "$ZULIP_PATH"
 fi
 
-"$ZULIP_PATH"/scripts/lib/install-node
+sudo "$ZULIP_PATH"/scripts/lib/install-node
```

Comment:
Hmm, this shouldn't be necessary -- this script should already be running as root. Lots of other steps will typically break if that isn't the case.

Are you finding this script actually running as non-root in some situation? If so, we should fix that.

Reply:
This is only required for circle so i can change this part.